pkg/specgen/specgen.go

```diff
@@ -602,14 +602,17 @@ type ContainerHealthCheckConfig struct {
 	// Requires that HealthConfig be set.
 	// Optional.
 	StartupHealthConfig *define.StartupHealthCheck `json:"startupHealthConfig,omitempty"`
-	// HealthLogDestination defines the destination where the log is stored
-	HealthLogDestination string `json:"healthLogDestination,omitempty"`
+	// HealthLogDestination defines the destination where the log is stored.
+	// Nil value means the default value (local).
+	HealthLogDestination string `json:"healthLogDestination"`
 	// HealthMaxLogCount is maximum number of attempts in the HealthCheck log file.
-	// ('0' value means an infinite number of attempts in the log file)
-	HealthMaxLogCount uint `json:"healthMaxLogCount,omitempty"`
+	// ('0' value means an infinite number of attempts in the log file).
+	// Nil value means the default value (5).
+	HealthMaxLogCount uint `json:"healthMaxLogCount"`
```

Comment:
also likely want to add a comment that omitempty should not be added and these should be converted to a pointer on the next major release

Reply:
Do we have something like this for employee tracking for the next major release, or is it just TODO?

Reply:
My personal syntax I use is `// TODO (6.0): ...`